packages/frontend/src/translations/en.global.json

```diff
@@ -1700,6 +1700,21 @@
             "title": "Enter Two-Factor Verification Code"
         }
     },
+    "twoFactorDisbleBanner": {
+        "desc": "NEAR wallet (wallet.near.org) will soon remove support for two-factor authentication, please disable 2FA for your connected accounts.",
+        "title": "${accountsCount} ${context} Your Attention",
+        "button": "Disable 2FA"
+    },
+    "twoFactorDisableLocked": {
+        "title": "Is your account locked?",
+        "descOne": "We couldn’t verify your 2FA method.",
```

Comment:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/25015977/192959481-5344f36a-33af-4af8-8190-efafb4b82238.png">

On this modal popup, we should say "We couldn't verify your 2FA method for **Account_id.**"

It is very hard for the user to know what account the bricked account is especially if they have multiple accounts.